src/screens/Profile/components/TariffPlanCard.tsx

```diff
@@ -17,6 +17,7 @@ const styles = makeStyles((theme: Theme) =>
 
 interface TariffPlanProps {
   tariffPlan: TariffPlanRecord;
+  readOnly?: boolean;
```

Comment:
I don't know if `readOnly` is the best name here. Maybe `editOnClick` might be better? Since in the other workflow when you click it you go to a screen to edit it.